community-version.py

```diff
@@ -1,17 +1,64 @@
 #!/usr/bin/python3
-## Community Version
-"""This is class SIMPLEcmd"""
+"""This is the SIMPLEcmd class for ASCII art conversion."""
 
 import os
+import sys
+
+import self
```

Comment:
What's the purpose of importing "self" here? It doesn't seem to being used for its intended purpose in the project

Reply:
for oop paradigm

Reply:
the 'self' pacakge is a decorator package which lets you do method chaining.

```python
 from self import self

 class CLS:
	@self
	def method(self):
		...

	@self
	def method2(self):
		...
```
```
>>> CLS().method().method2() # jQuery like chain
```

the use of **self** in an OOP sense does not require a package :)

You should be able to remove the package and not affect the functionality.